filterbuilder.go

```diff
@@ -196,7 +196,13 @@ func (f *FilterBuilder) RangeAdjacent(column, value string) *FilterBuilder {
 }
 
 func (f *FilterBuilder) Overlaps(column string, value []string) *FilterBuilder {
-	f.params[column] = "ov." + strings.Join(value, ",")
+	_new := []string{}
```

Comment:
just change here like "new" we not use '_' from variables and other sides looks good thanks for help

Reply:
K cool changed it to `newValue`